src/main/scala/awscala/dynamodbv2/AttributeValue.scala

```diff
@@ -37,6 +38,8 @@ object AttributeValue {
 
   def apply(v: aws.model.AttributeValue): AttributeValue = new AttributeValue(
     s = Option(v.getS),
+    //Unfortunately the ugly check is necessary here to bypass implicit conversion
+    bl = if (v.getBOOL != null) Option(v.getBOOL) else None,

```

Comment:
I think I found the solution, you need to first unbox to a primitive bool for the implicit conversion to work.

```
bl = Option[java.lang.Boolean](v.getBOOL).map(_.booleanValue()),
```